logging/logger.go

```
@@ -47,5 +51,17 @@ func SetupLogger(serviceName, version string) {
 	})
 
 	// Only log the info severity or above
-	log.SetLevel(log.InfoLevel)
+	if logLevel == "" {
+		logLevel = os.Getenv("SERVICE_LOG_LEVEL")
+	}
+	log.SetLevel(getLogrusLogLevel(strings.ToLower(logLevel)))
+}
+
+func getLogrusLogLevel(level string) log.Level {
+	lvl, err := log.ParseLevel(level)
+	if err != nil {
+		fmt.Printf("unknown log level %s, falling back to INFO level\n", level)
+		return log.InfoLevel
```

I think it would be cleaner to only fallback to `INFO` in case no log level was provided (neither in `SetupLogger`, nor `SERVICE_LOG_LEVEL`). In case an invalid log level is provided in either of these 2, we should return an error which could be converted to `log.Fatal` in each service.

This prevents us from having these kind of hidden logs on failure.

---

ok